packages/spec/integration/typescript/package.json

```diff
@@ -15,6 +15,7 @@
   },
   "dependencies": {
     "hops": "*",
-    "hops-typescript": "*"
+    "hops-typescript": "*",
+    "typescript": "^3.6.0"
```

Comment:
This doesn't make sense at the moment, because the spec packages are not workspaces.